config/cmake/README.txt.cmake.in

```diff
@@ -58,7 +67,7 @@ To test the installation with the examples;
 
 When executed, the ctest script will save the results to the log file, test.log, as
 indicated by the ctest command. If you wish the to see more build and test information,
-add "-VV" to the ctest command. The output should show near the end;
+add "-VV" to the ctest command. The output should show;
```

Comment:
Is ';' supposed to be ':'?

Reply:
It's been ";" forever, so I think it will be okay.